### PR TITLE
Skip setting test flags on failed srpm on empty tests

### PR DESCRIPTION
**Pull request by dhodovsk**

### CODE REVIEW

File: packit_service/worker/handler.py [hunk 1]
```diff
@@ -116,9 +111,9 @@ def report(
         self,
         state: str,
         description: str,
+        check_names: Union[str, list, None],
```
Comment (lachmanfrantisek) [inline packit_service/worker/handler.py]: Maybe more descriptive:

```suggestion
        check_names: Opotional[Union[str, list]],
```

File: packit_service/worker/github_handlers.py [hunk 2]
```diff
@@ -455,18 +455,18 @@ def run(self) -> HandlerResults:
...
 
                 r.report(
-                    "pending",
-                    "Tests are running ...",
-                    None,
-                    req.json()["url"],
+                    state="pending",
+                    description="Tests are running ...",
+                    build_id=None,
+                    url=req.json()["url"],
```
Comment (lachmanfrantisek) [inline packit_service/worker/github_handlers.py]: Thanks!

File: packit_service/worker/handler.py [hunk 3]
```diff
@@ -190,6 +183,8 @@ def report_rpm_build_failed_because_of_the_srpm_fail(self, build_check_names):
         )
 
     def report_tests_failed_because_of_the_build(self, test_check_names):
+        if not test_check_names:
+            return
```
Comment (lachmanfrantisek) [inline packit_service/worker/handler.py]: What about having this directly in the `report` method so every `report_*` method can use that?

**Comment by lachmanfrantisek** [comment]
> ACK if the failures are unrelated

Mypy has some problem with the code...

**Comment by TomasTomecek** [comment]
and needs rebase :mountain_cableway:

**Pull request closed without merging**